sdk/vaa/payloads.go

```diff
@@ -402,6 +410,16 @@ func (r BodyWormholeRelayerSetDefaultDeliveryProvider) Serialize() []byte {
 	return serializeBridgeGovernanceVaa(WormholeRelayerModuleStr, WormholeRelayerSetDefaultDeliveryProvider, r.ChainID, payload.Bytes())
 }
 
+func (r BodyGeneralPurposeGovernanceEvm) Serialize() []byte {
+	payload := &bytes.Buffer{}
+	payload.Write(r.GovernanceContract[:])
+	payload.Write(r.TargetContract[:])
+	// write payload len as uint16
+	MustWrite(payload, binary.BigEndian, uint16(len(r.Payload)))
+	payload.Write(r.Payload)
+	return serializeBridgeGovernanceVaa("GeneralPurposeGovernance", GovernanceAction(1), r.ChainID, payload.Bytes())
```

Comment:
Let's make `GeneralPurposeGovernance` a module string defined from a bytes32 array like the other module strings above?